pkg/statistics/handle/ddl/ddl.go

```diff
@@ -197,13 +164,46 @@ func (h *ddlHandlerImpl) HandleDDLEvent(t *util.DDLEvent) error {
 		}
 	case model.ActionFlashbackCluster:
 		return h.statsWriter.UpdateStatsVersion()
+	default:
+		logutil.StatsLogger().Info("Handle schema change event", zap.Stringer("event", t.SchemaChangeEvent))
```

Comment:
Since we already have a log above, we'll print two logs for each create table event. One of them is "none" and the other is the correct information.
Is that expected?

Reply:
https://github.com/pingcap/tidb/pull/55811/commits/249d1227d0b5b5d7e020aa6437473486fe9ee865